lib/package/components/input.css

```diff
@@ -0,0 +1,29 @@
+@charset "UTF-8";
+
+.form-input {
+  display: block;
+  width: 100%;
+  max-width: 100%;
+  line-height: 1.5;
+  padding: 0.375rem 0.75rem;
+  color: var(--gray-9);
+  background-color: var(--white);
+  border: 1px solid var(--gray-7);
+  border-radius: 0.25rem;
+}
+
+.form-label {
+  font-size: 1.2rem;
+}
+
+.has-error {
+  border: 2px solid var(--red-6);
+}
+
+.form-message {
+  margin-top: 0.25rem;
+  margin-bottom: 0.25rem;
+}
+.error-message {
```

Comment:
minor thing, you forgot to add space between declarations

Reply:
Thanks Arun. Will refactor it.